### PR TITLE
Docker, Command Line

### PR DESCRIPTION
**Pull request by joassouza**
Trying to make easier to review and to run locally. Next step is to create CI to run this code. I already have a task on my next sprint

### CODE REVIEW

File: ror-senior-dev/README.md [hunk 1]
```diff
@@ -69,7 +111,43 @@ have feature. So it is a bonus task & you can finish it if you have time.
...
+    "rate_currency": "USD"
+  }
+]
+```
+
+### Project Requirements
+
+1. Please, create one single branch for all the changes.
+2. Make sure your app run on docker and all the dependencies are included on it
+3. Create a private project and invite "joassouza", "yamanaltereh" once you're done
```
Comment (yamanaltereh) [inline ror-senior-dev/README.md]: @joassouza I do not like that 😂 
what is the point here? why not just keep uploaded a zip file with git history

Comment (joassouza) [reply ror-senior-dev/README.md]: the idea is to avoid lots of public repo where people can just copy and paste the solution

Comment (joassouza) [reply ror-senior-dev/README.md]: what do you suggest?

Comment (yamanaltereh) [reply ror-senior-dev/README.md]: no definitely not public, why not just send zip file including .git like usual?

Comment (joassouza) [reply ror-senior-dev/README.md]: usually they don't send the zip file. I upload the zip file on recruitee

**Review by yamanaltereh — APPROVED** [review]

**Pull request merged**